app/uk/gov/hmrc/nisp/views/includes/helpers/error_overview.scala.html

```diff
@@ -0,0 +1,16 @@
+@(form: Form[_], OptionalErrMsg:Option[String] = None)
+
+@if(form.hasErrors) {
+        <div id="error-summary-display" class="flash error-summary error-summary--show " role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@Messages("nisp.validation.summary.heading")</h2>
+            <ul class="js-error-summary-messages">
+                @form.errors.map { error =>
+                <li role="tooltip">
+                    <a href='#@error.key.replaceAll("""[\.\[\]]""", "_")' id='@error.key.replaceAll("""[\.\[\]]""", "_")-error-summary' data-focuses='@error.key.replaceAll("""[\.\[\]]""", "_")'>
```

Comment:
Because it wasn't working Ok with the textarea, I'm going to do double check on regex.
![error_old](https://cloud.githubusercontent.com/assets/14834785/17106076/dd8b4cf2-5281-11e6-84a1-52cffa602781.png)